bugbot/rules/component.py

```diff
@@ -91,6 +91,9 @@ def get_bugs(self, date="today", bug_ids=[]):
 
```

Comment:
I would add the logic to call the Fenix model here to avoid code duplicates.

Reply:
Done in aa18d50c9a5635fc36a2ee15a0ab4d7b38adb945